subsys/net/lib/sockets/sockets_tls.c

```diff
@@ -463,7 +541,15 @@ static struct tls_context *tls_alloc(void)
 
 	if (tls) {
 		k_sem_init(&tls->tls_established, 0, 1);
-
+#if defined(CONFIG_WOLFSSL)
+#if defined(CONFIG_NET_SOCKETS_ENABLE_DTLS)
+        tls->options.dtls_handshake_timeout_min = DTLS_TIMEOUT_DFL_MIN;
```

Comment:
Do we not have an API for settings the DTLS timeout and options? I think we do

Reply:
We do have APIs for it, and they are called in tls_wolfssl_set_dtls_timeouts(). This is for setting the initial default dtls timeouts to be set, which can be overridden with a setsockopt for TLS_DTLS_HANDSHAKE_TIMEOUT_MIN or TLS_DTLS_HANDSHAKE_TIMEOUT_MAX.